app/grants/models.py

```diff
@@ -69,7 +69,7 @@ class Subscription(SuperModel):
     amount_per_period = models.DecimalField(default=1, decimal_places=4, max_digits=50)
     token_address = models.CharField(max_length=255, default='0x0')
     gas_price = models.DecimalField(default=1, decimal_places=4, max_digits=50)
-    network = models.CharField(max_length=8, default='mainnet')
+    network = models.CharField(max_length=16, default='mainnet')
```

Comment:
No troll - which network will be 16 characters?

Reply:
testrpc has 13. I gave an extra 3 for flex. Can reduce to 13. But was getting errors during dev.

Reply:
Although docs say it defaults to current unix time on start. We could specify a network id when using ganache.

Reply:
I thought we used `local` or `testrpc` for `network` on other objects on the platform right now.  I don't think we need the full network name for the local testrpc instance here do we?

Reply:
Sure, we can have it as `local` or `testrpc`. That works.

Reply:
Reverted to 8 characters